src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildASelfContainedApp.cs

```diff
@@ -404,6 +404,32 @@ public void It_does_not_build_SelfContained_due_to_PublishSelfContained_being_tr
             outputDirectory.Should().NotHaveFile("hostfxr.dll"); // This file will only appear if SelfContained. 
         }
 
+        [Theory]
+        [InlineData("--p:PublishReadyToRun=true")]
+        [InlineData("-p:PublishSingleFile=true")]
+        [InlineData("-p:PublishSelfContained=true")]
+        [InlineData("-p:PublishAot=true")]
```

Comment:
Is there a reason the test is passing these as command line arguments instead of specifying them in the project?  It doesn't really make sense to pass these publish properties on the command line if you're just doing a `dotnet build`.  The expected scenario would be that they are in the project so when you do `dotnet publish` they take effect, but don't affect `dotnet build`.